src/scss/utils/text-stroke.scss

```diff
@@ -0,0 +1,50 @@
+// Mixin for internal use
+// prettier-ignore
+@mixin text-stroke($color, $width: 2px) {
+  overflow: initial;
+  text-shadow:
+    calc(-1 * $width) calc(-1 * $width) 0 $color,
```

Comment:
I have no idea what this does

Reply:
This is a mixin adaptation of [this answer from StackOverflow](https://stackoverflow.com/questions/4919076/outline-effect-to-text#4919231), which is basically a workaround of the experimental webkit property called [text-stroke](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke).